ee/server/services/account/Account.ts

```diff
@@ -56,12 +57,21 @@ const loginViaResume = async (resume: string): Promise<false | ILoginResult> =>
 	}
 
 	const { when } = user.services?.resume?.loginTokens?.find((token) => token.hashedToken === hashedToken) || {};
+	if (!when) {
+		throw new ClientSafeError(403, 'Your session has expired. Please log in again.');
+	}
+
+	const tokenExpires = _tokenExpiration(when);
+
+	if (new Date() >= tokenExpires) {
+		throw new ClientSafeError(403, 'Your session has expired. Please log in again.');
+	}
 
 	return {
 		uid: user._id,
 		token: resume,
-		hashedToken,
-		tokenExpires: when ? _tokenExpiration(when) : undefined,
+		hashedToken, // TODO should we return the hashed token? Meteor does not.
```

Comment:
Does it affect any other code if not return here? If not we should not return.

Reply:
I don't think it affects anything.. as stated, Meteor doesn't return that as well, so our clients definitely don't use it.. just need to make sure we don't use it **before** returning to client.

Reply:
That was my question :) if it's used somewhere else here. If not I think we should not return

Reply:
it is used on the logout method.. so I removed the TODO 😉